src/DataProtection/DataProtection/src/AuthenticatedEncryption/ManagedAuthenticatedEncryptorFactory.cs

```diff
@@ -96,14 +96,7 @@ private Func<SymmetricAlgorithm> GetSymmetricBlockCipherAlgorithmFactory(Managed
 
             if (configuration.EncryptionAlgorithmType == typeof(Aes))
             {
-                Func<Aes>? factory = null;
-                if (OSVersionUtil.IsWindows())
-                {
-                    // If we're on desktop CLR and running on Windows, use the FIPS-compliant implementation.
-                    factory = () => new AesCryptoServiceProvider();
-                }
-
-                return factory ?? Aes.Create;
+                return Aes.Create;
```

Comment:
Is the FIPS compliance statement no longer valid?

Reply:
AesCryptoServiceProvider is being obsoleted, not sure if the base API is FIPS compliant?

Reply:
Yup, but that's in netcore, no? Aes in net461 is probably not FIPs compliant. Do we need to cross-compile this?